types/src/v0/impls/solver.rs

```diff
@@ -0,0 +1,67 @@
+use committable::{Commitment, Committable};
+use hotshot::types::SignatureKey;
+
+use super::v0_3::{RollupRegistrationBody, RollupUpdatebody};
+
+impl Committable for RollupRegistrationBody {
+    fn tag() -> String {
+        "ROLLUP_REG".to_string()
```

Comment:
Unless we get some benefit from the shorter string, I think we should spell it out:

```suggestion
        "ROLLUP_REGISTRATION".to_string()
```